Documentation update: WPT data import; image links fixes (#116)
theodo/falco: docs/docs/getting-started/configuring-your-first-project.md

This email contains your WebPageTest API Key that we will use in the next section.

## Import data from WPT

> WPT service provides numerouse options for running your tests ( location, bandwidth ), so you **must** import this data into your setup. To do this, via admin interface:

Reviewer: ```suggestion
WPT provides numerous options for running your tests (location, bandwidth), so you **must** import this data before setting up your first project. You can do this via the admin interface:
```

> 1. head to `<your Falco instance URL>/admin/django_celery_beat/periodictask/` 

Reviewer: ```suggestion
1. Head to `<your Falco instance URL>/admin/django_celery_beat/periodictask/` 
```

> 1. mark checkbox for `Get all wpt audit configurations: 0 1 * * * (m/h/d/dM/MY) UTC` 

Reviewer: ```suggestion
1. Check the `Get all wpt audit configurations: 0 1 * * * (m/h/d/dM/MY) UTC` checkbox
```

> 1. from combobox (dropdown list) choose to Run the task

Reviewer: ```suggestion
1. Choose the Run the task option from the dropdown list
```

![Run import task](/docs/static/img/wpt-update.png)

## Creating your first project

To create your first project, head to `<your Falco instance URL>/admin/` and connect to the admin interface. Then click on the `+ Add` button in the `Projects` section, and you should see the following form:

> ![The form used to create a project](/docs/static/img/create-project.png)

Reviewer: Are you sure about this?

When I try accessing https://getfal.co/img/create-project.png it works as intended, but if I go to https://getfal.co/docs/static/img/create-project.png it gives a Docusaurus 404.

Author: I was checking with Github's preview. Doesn't it mean that dir structure here and at getfal.co differs?

Reviewer: My guess is that [Docusaurus](https://v2.docusaurus.io/), the docs framework we use for Falco, doesn’t exactly mimic the URL structure with the file structure with regards to images / static assets.

Author: well, i cannot check it in advance, so no problem from my side to keep url as it was

Reviewer: Great then! Thanks!

> ![The form used to create a project](/docs/static/img/create-project.png)

Reviewer: ```suggestion
![The form used to create a project](/img/create-project.png)
```
